modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm

```diff
@@ -150,7 +150,7 @@
 	new /obj/item/radio/headset/heads/nanotrasen_consultant(src)
 	new /obj/item/storage/photo_album/personal(src)
 	new /obj/item/bedsheet/centcom(src)
-	new /obj/item/storage/bag/garment/nanotrasen_consultant(src)
+	new /obj/item/storage/bag/garment/zubbers/nanotrasen_consultant(src) //Bubberstation Edit
```

Comment:
You could just add to the original bag in a way like....


`/obj/item/storage/bag/garment/nanotrasen_consultant
	. = ..()`
	
	Then the items.

Reply:
> You could just add to the original bag in a way like....
> 
> `/obj/item/storage/bag/garment/nanotrasen_consultant . = ..()`
> 
> ```
> Then the items.
> ```

Well, I guess that's true. But it was also to add my own things, but sorting them together instead of leaving it a massive mess.